actionmailer/test/test_helper_test.rb

```diff
@@ -17,6 +17,12 @@ def test_args(recipient, name)
       from: "tester@example.com"
   end
 
+  def test_named_args(recipient:, name:)
```

Comment:
I noticed this scenario wasn't accounted for. With recent rubies the named attributes seem like a better approach.

Reply:
Another nice addition!